db/migrate/20190924195715_add_ial_to_service_provider_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddIalToServiceProviderRequests < ActiveRecord::Migration[5.1]
+  def change
+    add_column :service_provider_requests, :ial, :string
```

Comment:
@jmhooper During our PR (Me, Clara and Steve) review we discovered that the service_provider_requests table is probably 20+GB. Therefore we see no clear path forward to alter and populate the new column on this table.  We would like to update the AC's to allow for this column to remain loa.